curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-to-pass-an-object-as-a-functions-parameters.english.md

```diff
@@ -75,6 +74,8 @@ console.log(half(stats)); // should be 28.015
 <section id='solution'>
 
 ```js
-// solution required
+function half({max, min}) {
```

Comment:
@ezioda004 solution should contain a seed code.

Reply:
Ahh fixed, thanks!